git-hooks/pre-commit

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+python3 lean2md.py lean _posts
```

Comment:
This is nice. We also need `git add _posts` right after that so the changes will be committed. Does it work?

Reply:
Oh I just tested. It works!
```suggestion
python3 lean2md.py lean _posts
git add _posts
```